Resources/Prototypes/_EinsteinEngines/NPCs/carp.yml

```
@@ -0,0 +1,82 @@
+        Alive:
+          Base: alive
+          BaseUnshaded: mouth
+        Critical:
+          Base: critical
+          BaseUnshaded: critical_mouth
+        Dead:
+          Base: dead
+          BaseUnshaded: dead_mouth
+    - type: RandomSprite
+      available:
+      - enum.DamageStateVisualLayers.Base:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)